src/uu/mv/src/mv.rs

```diff
@@ -334,10 +318,11 @@ fn handle_two_paths(source: &Path, target: &Path, opts: &Options) -> UResult<()>
         }
     }
 
-    let target_is_dir = target.is_dir();
+    let target_is_dir: bool = target.is_dir();
+    let source_is_dir: bool = source.is_dir();
```

Comment:
It's obvious that the vars are `bool`.
```suggestion
    let target_is_dir = target.is_dir();
    let source_is_dir = source.is_dir();
```